File: LeetCode/Majority Element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        s1 = set(nums)
        max = -111111
        for i in s1:
            if (nums.count(i)>max):
                max = nums.count(i)
                el = i
        return el

File: LeetCode/Majority Element.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
        for(int i=0; i<nums.length; i++){
            map.put(nums[i], map.getOrDefault(nums[i],0)+1);
        }
        
        int val = (int)Math.floor(nums.length/2);
        for(int k:map.keySet()){
            if(map.get(k)>val) return k;
        }
        return 0;
    }
}
